java/integTest/build.gradle

```diff
@@ -10,6 +10,8 @@ dependencies {
     // client
     implementation project(':client')
 
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.13.0'
```

Comment:
why is it necessary for SELECT?

Reply:
This helps up manipulate arrays which is part of the IT tests. 
see: ArrayUtils